### PR TITLE
Modify script that converts RCTC xlsx to readable csv

### PR DESCRIPTION
**Pull request by angelathe**
# PULL REQUEST

## Summary
Modifies script to also convert and output Expansion List from RCTC xlsx

## Related Issue
Adds additional fix to #1149 

## Additional Information
Anything else the review team should know?

## Checklist

- [ ] If this code affects the other scrum team, have they been notified? (In Slack, as reviewers, etc.)

[//]: # (PR title: Remember to name your PR descriptively!)

### CODE REVIEW

File: containers/fhir-converter/rctc-to-json.py [hunk 1]
```diff
@@ -6,16 +6,24 @@
 
 # arg list: 1) input file location/name 2) output file location/name
 wb = openpyxl.load_workbook(sys.argv[1])
-jsonData = {}
+jsonData_grouping = {}
+jsonData_expansion = {}
+top_rows_buffer = 17
```
Comment (nickbristow) [inline containers/fhir-converter/rctc-to-json.py:11]: just a question, what are these hard coded numbers?

Comment (angelathe) [reply containers/fhir-converter/rctc-to-json.py:11]: @nickbristow In the core table with the RCTC codes (ex. [Link](https://ecr.aimsplatform.org/cms/resources/blocks/reference-rctc-3-22.xlsx)) there are a couple misc. rows before the actual table data begins. So `top_rows_buffer` indicates that there are 17 rows before the Grouping Data table, and `tables_buffer` represents the number of rows between the Grouping Data table and the Expansion Data table. 

Values seem to be consistent (so far) between table updates, but lmk if you think we should either 1) write a comment to clarify this, or 2) find the beginning rows of the tables without using hard-coded vals!

Comment (angelathe) [reply containers/fhir-converter/rctc-to-json.py]: fyi, added comments explaining what these hard-coded vals are

**Review by JNygaard-Skylight — COMMENTED** [review]
My only suggestion is to look into using the [`argparse` library](https://docs.python.org/3.10/library/argparse.html). It will make validating the inputs easier, and will add documentation to the command line. But everything as is looks good to me, so I'll leave it up to you if you want to look into adding `argparse`.

**Review by JNygaard-Skylight — APPROVED** [review]
LGTM!

**Pull request merged**